services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -193,6 +202,20 @@ public Response getMainPage() throws IOException, FreeMarkerException {
         return builder.build();
     }
 
+    private Response redirectToLogin(String path) {
+        UriBuilder consoleUriBuilder = Urls.accountBase(session.getContext().getUri().getBaseUri());
+        if (!StringUtil.isNullOrEmpty(path)) {
+            consoleUriBuilder.path(path);
+        }
+        URI targetUri = consoleUriBuilder.build(realm.getName());
+
+        var oauthRedirect = new AbstractSecuredLocalService.OAuthRedirect();
+        oauthRedirect.setAuthUrl(OIDCLoginProtocolService.authUrl(session.getContext().getUri()).build(realm.getName()).toString());
```

Comment:
To avoid multiple sources for the client config, can we use a public endpoint to fetch the config? IIRC, we had this before, don't we?